django-rgd-imagery/tests/data_fixtures.py

```diff
@@ -132,14 +134,24 @@ def sample_raster_url():
         'LC08_L1TP_034032_20200429_20200509_01_T1_sr_band2.tif',
         'LC08_L1TP_034032_20200429_20200509_01_T1_sr_band3.tif',
     ]
-    images = [
-        factories.ImageFactory(
+    band_ranges = [
+        (Decimal(0.75), Decimal(1.00)),  # "nir"
+        (Decimal(0.70), Decimal(0.79)),  # "rededge"
+        (Decimal(0.41), Decimal(0.43)),  # not a common name
+    ]
+    images = []
+    for i, (f, band_range) in enumerate(zip(landsat_files, band_ranges)):
+        image = factories.ImageFactory(
             file__type=FileSourceType.URL,
             file__file=None,
             file__url=datastore.get_url(f),
         )
-        for f in landsat_files
-    ]
+        images.append(image)
+        factories.BandMetaFactory(
+            parent_image=image,
+            band_range=band_range,
+            band_number=(i + 1),
+        )
```

Comment:
> This test isn't a realistic use case as there are two `(i+1)` `BandMetas` here.

To be sure we're on the same page: this creates three `BandMeta` objects with `band_number`s of 1, 2, and 3. Each `BandMeta` creates special edge-cases to test (mainly in `band_ranges`).

> Anywhere we use the `Meta` suffix, we ought not manually create these records. `*Meta` models should all be managed by the ETL routines.

This makes sense, but it seems like it would be difficult to manufacture all the edge-cases that directly creating `BandMeta` has allowed. Is there an easy way to create an `Image` via `ImageFactory` with the band information here?

Reply:
> but it seems like it would be difficult to manufacture all the edge-cases that directly creating BandMeta has allowed

definitely difficult... 

> Is there an easy way to create an Image via ImageFactory with the band information here?

No, not really... because the ETL routines populate these. Perhaps the best thing to do here is just to update the existing `BandMeta` records rather than creating new ones.

Reply:
I've gone ahead and added this change. There's really no good way to do this at the moment because the ETL routines do not populate all of the fields on the `BandMeta` model. We can come back to this if we need to.